src/main/java/com/emc/pravega/perf/PravegaPerfTest.java

```diff
@@ -400,76 +602,70 @@ public void run() {
                 return null;
 
           };
-        }
-    }
+      }
+}
 
-    /**
-     * A Sensor reader class that reads the temperative data
-     */
-    private static class ReaderWorker implements Runnable {
-       @Setter
-        public static AtomicInteger totalEvents;
-        private EventStreamReader<String> reader;
-        String readerId;
+class ReaderWorker implements Callable<Void> {
```

Comment:
static?

Reply:
Probably private too right?

Reply:
yes, total Events is static variable accessed my multiple readers

Reply:
Please move this into it's own file.

Reply:
yes , its done